### PR TITLE
Added the view_proteinstructures viewer spec/display files

### PR DESCRIPTION
**Pull request by qzzhang**

### CODE REVIEW

File: ui/narrative/methods/view_proteinstructures/spec.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{
+  "name" : "View ProteinStructures",
+  "ver" : "1.0.0",
+  "authors" : [ ],
+  "contact" : "http://kbase.us/contact-us/",
```
Comment (briehl) [inline ui/narrative/methods/view_proteinstructures/spec.json]: That page isn't found, please set it to https://www.kbase.us/support/

File: ui/narrative/methods/view_proteinstructures/spec.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  "visble" : true,
```
Comment (briehl) [inline ui/narrative/methods/view_proteinstructures/spec.json]: typo: `visible`

File: ui/narrative/methods/view_proteinstructures/display.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+        related :
+            []
+        next :
+            []
+
+#
+# Configure the display and description of the parameters
+#
+parameters :
+    param0 :
```
Comment (briehl) [inline ui/narrative/methods/view_proteinstructures/display.yaml]: There is no `param0` in the spec file. This should be `proteinstructures`.

File: ui/narrative/methods/view_proteinstructures/spec.json [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+  "categories" : ["active", "viewers"],
+  "app_type" : "viewer",
+  "widgets" : {
+    "input" : null,
+    "output" : "KBaseProteinStructuresView"
```
Comment (briehl) [inline ui/narrative/methods/view_proteinstructures/spec.json:11]: Note that this widget doesn't exist in the Narrative yet. Still fine to merge, though.

**Review by briehl — CHANGES_REQUESTED** [review]
A couple comments, and a couple things to change.

Also, update the version in `kbase.yml` to `1.1.0` to reflect adding new apps.

File: ui/narrative/methods/view_proteinstructures/spec.json (hunk 1 above)
Comment (qzzhang) [reply ui/narrative/methods/view_proteinstructures/spec.json]: Done.

Comment (qzzhang) [reply ui/narrative/methods/view_proteinstructures/spec.json]: I have found 57 cases of `contact: "help@kbase.us"` in addition to several `contact: "[help@kbase.us](http://kbase.us/contact-us/)"` when I was correcting the spelling of `visble`.

File: ui/narrative/methods/view_proteinstructures/display.yaml (hunk 3 above)
Comment (qzzhang) [reply ui/narrative/methods/view_proteinstructures/display.yaml]: Yes.

File: ui/narrative/methods/view_proteinstructures/spec.json (hunk 4 above)
Comment (qzzhang) [reply ui/narrative/methods/view_proteinstructures/spec.json:11]: Good.  I am developing the widget using the local narrative installation.  For now I need the NarrativeViewers and narrative_method_specs_ci to help me with the development.

Comment (qzzhang) [reply ui/narrative/methods/view_proteinstructures/spec.json:11]: If the "input" in kept as `null`, would it prevent me from testing my widget locally?

File: ui/narrative/methods/view_proteinstructures/spec.json (hunk 2 above)
Comment (qzzhang) [reply ui/narrative/methods/view_proteinstructures/spec.json]: This is an embarrassing error--obviously we have been copy/pasting from existing examples.  I found 53 of `visble` in the module.  So I decide to change all of them with a hope that my change won't cause any interruption.

Comment (briehl) [reply ui/narrative/methods/view_proteinstructures/spec.json]: Wow! Yeah, that's a good find.

File: ui/narrative/methods/view_proteinstructures/spec.json (hunk 4 above)
Comment (briehl) [reply ui/narrative/methods/view_proteinstructures/spec.json:11]: No, that doesn't have any effect.

**Review by briehl — CHANGES_REQUESTED** [review]
Please bump the version in kbase.yml to 1.1.0 and it'll be ready.

**Review by briehl — APPROVED** [review]

**Pull request merged**